index.html

```diff
@@ -166,20 +165,43 @@ <h3><dfn>CropTarget</dfn> Definition</h3>
           <li>
             <p>Let <var>cropTarget</var> be a new object of type {{CropTarget}}.</p>
           </li>
+          <li> <p>Let <var>weakRef</var> be a weak reference to <var>element</var>.</p></li>
           <li>
-            <p>Let <var>weakRef</var> be a weak reference to <var>element</var>.</p>
-            <p>
-              [=Create a CropTarget|Create=] <var>cropTarget</var>.[[\Element]] initialized to
-              <var>weakRef</var>.
-            </p>
+            <p> Set <var>cropTarget</var>.[[\Element]] to <var>weakRef</var>. </p>
             <div class="note">
               <p>
                 <var>cropTarget</var> keeps a weak reference to the element it represents. In other
                 words, <var>cropTarget</var> will not prevent garbage collection of its element.
               </p>
             </div>
           </li>
+          <li>
+            <p>Set |cropTarget|.[[\top-level browsing context]] to [=this=]'s [=browsing context=]'s
+             [=top-level browsing context=].</p>
+          </li>
+          <li>
+            <p>Let |promise| be [=a new promise=].</p>
+          </li>
+          <li>
+            <p>Run the following steps in parallel:</p>
+            <ol>
+              <li><p>In an [=implementation-defined=] way, propagate state associated with |cropTarget|.</p></li>
+              <li>
+                <p>If state propagation was successful, [=resolve=] |promise|. Otherwise, [=reject=] |promise|.</p>
```

Comment:
This is under heavy discussion in #11 and there's no concensus to merge this. Web developers aren't going to expect this to fail, and failures are going to be surpassing and browser dependent, which is bad for web compat.

This also cements an inferior approach to this problem, which I've argued is actually slower than the alternative, a claim no one has challenged yet.

Reply:
> This is under heavy discussion in #17

Please note the PR's description.

> Web developers aren't going to expect this to fail

Why?

> This also cements an inferior approach to this problem

https://www.w3.org/Consortium/cepc/#expected-behavior

> which I've argued is actually slower than the alternative, a claim no one has challenged yet.

https://github.com/w3c/mediacapture-region/issues/17#issuecomment-1131846988

Reply:
https://github.com/w3c/mediacapture-region/issues/48 is where this discussion is happening. Please comment there.